packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -43,8 +37,8 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
     authorsImageUrls: (string | undefined)[];
   };
 
-  export type Author = {
-    key?: string; // TODO temporary, need refactor
+  export type AuthorAttributes = {
+    key: string | null; // TODO temporary, need refactor
```

Comment:
It's not really an attribute, and we already have a key in `Author` type so it would lead to duplicated declaration

We need `key?: string` in the blog front matter author instead since in this case the key can be absent from front matter (which will actually be the case for 99% blog posts)